package.bzl

```diff
@@ -0,0 +1,43 @@
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Package file which defines build_bazel_rules_nodejs version in skylark
+
+check_rules_nodejs_version can be used in downstream WORKSPACES to check
+against a minimum dependent build_bazel_rules_nodejs version.
+"""
+
+load("//internal/common:check_version.bzl", "check_version")
+
+VERSION = "0.11.2"
```

Comment:
1) comment here saying it needs to be synced with package.json
2) edit the end of the README.md so I know how to do the next release
3) where is todo/issue reminding us to fix the duplication?